### ISSUE TITLE
Behavior on unknown chainslug

### ISSUE THREAD

**Issue by vaibhavchellani** [SocketDotTech/socket-DL#258]
We cannot assume a socket is aware of all other socket's so it makes sense to not maintain any kind of data store that maintains all other socket's as it would make it really hard to be permissionless, it should be possible that anyone can deploy a socket on a new chain, wire to it and get going

However, what is the desired behaviour if the outbound is called with a toChainSlug which hasn't been configured? I am not sure if we fail silently or revert rn

**Comment by ameeshaagrawal** [comment]
Currently socket stores the plug config for each `toChainSlug` which is used at outbound to get the capacitor address which should be used to queue messages in the packet going to be sealed. 

If there is no switchboard for `toChainSlug`, plugs cannot register for that slug and hence call will revert.
If someone registers the switchboard for this unsupported `toChainSlug`, plugs will be able to register and hence `outbound` call will go through. But the transmission will not happen.

**Comment by arthcp** [comment]
I think plugs cannot connect in case there is no capacitor. So unless someone connects a switchboard to some remote chain, plugs cannot send messages.

**Comment by ameeshaagrawal** [comment]
Only if transmitter has the role. Transmitters are chain specific, so they can only seal/propose if we have grant roles for this new `toChainSlug`.

**Comment by vaibhavchellani** [comment]
makes sense, I can totally imagine a transmitManager has an open function that allows anyone to seal for unregistered destChainSlug
